AudioDeviceExample/AudioDevices/ExampleAVAudioEngineDevice.m

```diff
@@ -52,6 +52,12 @@
 
     // Core Audio's VoiceProcessingIO audio unit.
     AudioUnit audioUnit;
+
+    /*
+     * Points to AVAudioEngine's manualRenderingBlock. This block is called from within the VoiceProcessingIO playout
+     * callback in order to receive mixed audio data from AVAudioEngine in real time.
+     */
+    void *renderBlock;
```

Comment:
Nice docs. Is there another type besides void * that we can use?

Reply:
let me check this.

Reply:
It is possible to use `AVAudioEngineManualRenderingBlock` but is it OK to use a block from CoreAudio ? 🤔

Reply:
I think we should use the correct type if possible. I'm not sure about the real time safety of calling a block from a thread though.

Reply:
As discussed, ARC may interfere here with AVAudioEngineManualRenderingBlock. Keeping the code as is.